crates/testing/tests/tests_1/test_success.rs

```diff
@@ -39,6 +42,31 @@ cross_tests!(
     },
 );
 
+cross_tests!(
+    TestName: test_success_marketplace,
+    Impls: [MemoryImpl],
+    Types: [TestTypes],
+    Versions: [MarketplaceUpgradeTestVersions],
+    Ignore: false,
+    Metadata: {
+        let mut metadata = TestDescription {
+            // allow more time to pass in CI
+            completion_task_description: CompletionTaskDescription::TimeBasedCompletionTaskBuilder(
+                                             TimeBasedCompletionTaskDescription {
+                                                 duration: Duration::from_secs(60),
+                                             },
+                                         ),
+            upgrade_view: Some(5),
+            ..TestDescription::default()
+        };
+
+
+        metadata.overall_safety_properties.num_failed_views = 10;
```

Comment:
Why does it need to allow 10 failed views?

Reply:
oops -- fixed!